pkgs/top-level/python-aliases.nix

```diff
@@ -516,6 +516,7 @@ mapAliases ({
   ptable = throw "ptable has been removed, as it is unmaintained and broken"; # added 2025-05-25
   pur = throw "pur has been renamed to pkgs.pur"; # added 2021-11-08
   pushbullet = pushbullet-py; # Added 2022-10-15
+  pulsar = pulsar-client; # Added 2025-05-28
```

Comment:
I don't think this should even exist. https://pypi.org/project/pulsar/ is a different package

Reply:
Sounds good. Dropped.